src/os/bluestore/bluestore_types.cc

```diff
@@ -1092,7 +1092,7 @@ bool bluestore_blob_t::release_extents(bool all,
       ++pext_it;
     }
     else {
-      //assert(pext_loffs == pext_loffs_start);
+      //ceph_assert(pext_loffs == pext_loffs_start);
```

Comment:
This one is replaced by `ceph_assert(delta0 >= 0)` below (i.e. `pext_loffs` is allowed to be greater than `pext_loffs_start` now).